src/native/corehost/fxr/sdk_info.cpp

```diff
@@ -103,13 +103,13 @@ void sdk_info::get_all_sdk_infos(
     std::sort(sdk_infos->begin(), sdk_infos->end(), compare_by_version_ascending_then_hive_depth_descending);
 }
 
-/*static*/ bool sdk_info::print_all_sdks(const pal::string_t& dotnet_dir, const pal::string_t& leading_whitespace)
+/*static*/ bool sdk_info::print_all_sdks(const pal::string_t& dotnet_dir, const pal::char_t* leading_whitespace)
```

Comment:
I'm fine with converting this to avoid an allocation. Can we assert these are non-null in both of these entry functions?